_data/changelogs/component-form.yml

```diff
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added guidance discouraging most uses of disabled form inputs.
+    summaryAdditional: We also added guidance on the rare instances when disabled inputs improve usability. 
+    affectsAccessibility: true
```

Comment:
**suggestion:** Recommend we remove the "affectsAccessibility" tag here because the change itself does not affect the accessibility of the content. 

```suggestion
```